tests/virtualization/vagrant/boxes/tumbleweed.pm

```diff
@@ -46,12 +45,13 @@ sub run() {
     $arch_ext = "_$arch" if !is_x86_64();
     my $build = get_required_var('BUILD');
 
-    my %boxes = (
-        # Tumbleweed.x86_64-1.0-{libvirt|virtualbox}-Snapshot20190704.vagrant.{libvirt|virtualbox}.box
-        libvirt => "$version.$arch-1.0-libvirt$arch_ext-Snapshot$build.vagrant.libvirt.box",
-    );
-    # virtualbox is supported only on x86_64
-    %boxes = (%boxes, virtualbox => "$version.$arch-1.0-virtualbox-Snapshot$build.vagrant.virtualbox.box") if $is_virtualbox_applicable;
+    my %boxes = ();
+    if ($is_virtualbox_applicable) {
+        my %boxes = (
```

Comment:
This doesn't assign but creates a new variable

Reply:
Should be fixed now